### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by aqsaaqeel**
Listing products using api.

Checkout : 
- wishlist-page
- cart-page

[thrifterssstop.netlify.app](https://thrifterssstop.netlify.app/)

In next PR, will add images for all products and cart management

### CODE REVIEW

File: src/Pages/cart-page.jsx [hunk 1]
```diff
@@ -16,14 +18,11 @@ function CartPage() {
...
-                    <VerticalCard />
-                    <VerticalCard />
-                    <VerticalCard />
-                    <VerticalCard />
-                    <VerticalCard />
-                    <VerticalCard />
-                    <VerticalCard />
+                    { data && data.map(item => {
+                        return(
+                            <VerticalCard key = {item.id} name = {item.name} price = {item.price}/>
```
Comment (rehanmohiuddin) [inline src/Pages/cart-page.jsx]: pass whole item in the props then extract in that component

File: src/Pages/landing-page.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+                    <Card collection="Vibrant Collection" />
+
+                    <Card collection="Evergreen Collection" />
+                </div>
+
+                <Heading name="Categories" />
+
+                <div className="category-container">
+                    <SmallCard name="Men's Apparels" url={require("../assets/images/man-shirt.jpg")} />
```
Comment (rehanmohiuddin) [inline src/Pages/landing-page.jsx:28]: dont use require instead at top of the file do this import demoImg from "../assets/imgName.png" put this demoImg in img src attribute

**Review by rehanmohiuddin — APPROVED** [review]
Use Prettier for code formatting
Suggested some changes

**Comment by aqsaaqeel** [comment]
> Use Prettier for code formatting Suggested some changes
 Thank you, making the changes

**Review by prathmeshjagtap — APPROVED** [review]

**Pull request merged**